src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let price = 0;
```

Comment:
checklist p2

> [CODE STYLE] - Avoid reassigning variables values. It's better to use const instead of let.

Reply:
@ruslanpashkov price changed accord the days, it doesn't work with const variable

Reply:
@ruslanpashkov fixed the loop (recommended above) and I got what you meant :)